index.js

```diff
@@ -9,9 +9,18 @@ const kGenerateCallbackUriParams = Symbol.for('fastify-oauth2.generate-callback-
 const { promisify, callbackify } = require('node:util')
 
 const USER_AGENT = 'fastify-oauth2'
+const VERIFIER_COOKIE_NAME = 'oauth2-code-verifier'
+const S256 = 'S256'
+const PKCE_METHODS = ['plain', S256]
```

Comment:
I would recommend to list S256 first and somehow note that `plain` use is not recommended and only intended for applications connecting to legacy servers. Best case it would not be implemented anymore at all.